cmd/server/server.go

```diff
@@ -420,32 +407,34 @@ func Initialize(conf config.Provider) error {
 	}
 	replayValidator := job.NewReplayValidator(models.Scheduler)
 	replaySyncer := job.NewReplaySyncer(
+		l,
 		replaySpecRepoFac,
 		projectRepoFac,
 		models.Scheduler,
 		func() time.Time {
 			return time.Now().UTC()
 		},
 	)
-	replayManager := job.NewManager(replayWorkerFactory, replaySpecRepoFac, utils.NewUUIDProvider(), job.ReplayManagerConfig{
+	replayManager := job.NewManager(l, replayWorkerFactory, replaySpecRepoFac, utils.NewUUIDProvider(), job.ReplayManagerConfig{
 		NumWorkers:    conf.GetServe().ReplayNumWorkers,
 		WorkerTimeout: conf.GetServe().ReplayWorkerTimeoutSecs,
 		RunTimeout:    conf.GetServe().ReplayRunTimeoutSecs,
 	}, models.Scheduler, replayValidator, replaySyncer)
 
 	notificationContext, cancelNotifiers := context.WithCancel(context.Background())
 	defer cancelNotifiers()
-	eventService := job.NewEventService(map[string]models.Notifier{
+	eventService := job.NewEventService(l, map[string]models.Notifier{
 		"slack": slack.NewNotifier(notificationContext, slackapi.APIURL,
 			slack.DefaultEventBatchInterval,
 			func(err error) {
-				logger.E(err)
+				l.Error("Error: No notification from slack", err)
```

Comment:
`Error: No notification from slack` this is not a valid error statement here, maybe we can have
```go
l.Error("slack error accumulator", "error", err)
```